main.go

```diff
@@ -65,6 +65,11 @@ var (
 		"icmp": prober.ProbeICMP,
 		"dns":  prober.ProbeDNS,
 	}
+
+	probeModuleUnknownCounter = prometheus.NewCounter(prometheus.CounterOpts{
+		Name: "probe_module_unknown",
```

Comment:
Counters should end with _total.

This is not a metric from a probe, it is a metric about the blackbox exporter. Thus it should be blackbox_